MotionMark/tests/resources/main.js

```diff
@@ -155,6 +155,29 @@ Controller = Utilities.createClass(
         return averageFrameLength;
     },
 
+    _getFrameType: function(samples, i)
+    {
+        return samples[sampleTypeIndex][i];
+    },
+
+    _getComplexity: function(samples, i)
+    {
+        return samples[sampleComplexityIndex][i];
+    },
+
+    _getFrameLength: function(samples, i)
+    {
+        return samples[sampleTimeIndex][i] - samples[sampleTimeIndex][i - 1];
+    },
+    
+    _previousFrameComplexity: function(samples, currentSampleCount)
```

Comment:
Can't `currentSampleCount` be just `i` like other functions? Using `currentSampleCount` gives an impression that this function can be used for the last sample only.